src/infiniteGrid.js

```diff
@@ -406,11 +406,14 @@ eg.module("infiniteGrid", ["jQuery", eg, window, document, "Outlayer"], function
 		 * @return {eg.InfiniteGrid} instance of itself<ko>자신의 인스턴스</ko>
 		 */
 		setStatus: function(status) {
-			this.core.element.style.cssText = status.cssText;
-			this.core.$element.html(status.html);
+			if (!status || $.isEmptyObject(status)) {
+				return this;
+			}
+			status.cssText && (this.core.element.style.cssText = status.cssText);
```

Comment:
If status object is provided by "getStatus" method, it always has all properties(cssText, html, core, data). But the "setStatus" method is public API, parameter can be anything. 

It is necessary to check each property in parameter If we need every properties.